addons/sale_expense/models/hr_expense_split.py

```diff
@@ -12,9 +11,11 @@ def default_get(self, fields):
         if 'expense_id' in result:
             expense = self.env['hr.expense'].browse(result['expense_id'])
             result['sale_order_id'] = expense.sale_order_id
+            result['sale_order_line_id'] = expense.sale_order_line_id
```

Comment:
Why not let the file be computed?
`default_get` is dirty and useless most of the time.

The (almost) only way it should be used is to interpret the context in a different way than the default.